<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MagnetMenu</class>
 <widget class="QMainWindow" name="MagnetMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="TimeLabel" name="currentTime">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>520</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>1234567890</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="sreachButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>180</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>搜索模式</string>
    </property>
   </widget>
   <widget class="QPushButton" name="simpleButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>280</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>简单调查模式</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mappedButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>360</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>差分定位模式</string>
    </property>
   </widget>
   <widget class="QPushButton" name="baseButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>325</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>base station</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reviewButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>430</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>data review</string>
    </property>
   </widget>
   <widget class="QPushButton" name="transferButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>470</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>数据管理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setupButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>400</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-background-color: rgb(255, 255, 127);</string>
    </property>
    <property name="text">
     <string>system setup</string>
    </property>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>-10</y>
      <width>261</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;Times New Roman&quot;;
</string>
    </property>
    <property name="text">
     <string>MAGNETOMETER</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="controlHint">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>50</y>
      <width>331</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>22</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 22pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Use arrow keys to select desired function.                                                                                                                 Confirm with &quot;ENTER&quot;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="DigitalClock" name="lcdTimeLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>200</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="drawButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>400</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>绘图调查模式</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimeLabel</class>
   <extends>QLabel</extends>
   <header>timelabel.h</header>
  </customwidget>
  <customwidget>
   <class>DigitalClock</class>
   <extends>QLCDNumber</extends>
   <header location="global">digitalclock.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>sreachButton</tabstop>
  <tabstop>simpleButton</tabstop>
  <tabstop>drawButton</tabstop>
  <tabstop>mappedButton</tabstop>
  <tabstop>baseButton</tabstop>
  <tabstop>transferButton</tabstop>
  <tabstop>reviewButton</tabstop>
  <tabstop>setupButton</tabstop>
  <tabstop>lcdTimeLabel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
